content/english/android-tictactoe/variables.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Variables"
+date: 2020-07-27T14:17:07.42-07:00
+draft: true
+weight: 10
+---
+
+# Variables
+
+Variables are simply names that we can give to values such as strings, numbers, and booleans. We use variables to store information so the computer can keep track of it. Variables are created by using the type of the variable and the name of the variable. Then you can assign that variable to the information that you want to store by using `=` and the content.
+
+Look at the variables at the top of of `MainActivity.kt`.
+
+```java
+    var Player1 = ArrayList<Int>()
```

Comment:
We'll need a basics page for data structures (ArrayList etc.) but I think that's already being take care of by someone else.